content/issues/95.markdown

```diff
@@ -0,0 +1,91 @@
+<!-- 2018-02-22 -->
+
+Welcome to another issue of Haskell Weekly!
+[Haskell](https://haskell-lang.org) is a safe, purely functional programming language with a fast, concurrent runtime.
+This is a weekly summary of what's going on in its community.
+
+Want to see something featured in Haskell Weekly?
+Open an issue or pull request [on GitHub](https://github.com/haskellweekly/haskellweekly.github.io).
+
+## Featured
+
+-   [GSoC 2018 ideas](https://summer.haskell.org/ideas.html)
+
+    > This is a list of ideas for students who are considering to apply to Google Summer of Code 2018 for Haskell.org.
+
+-   [Haskell Symposium 2018](https://www.haskell.org/haskell-symposium/2018/)
+
+    > The Haskell Symposium aims to present original research on Haskell, discuss practical experience and future development of the language, and to promote other forms of denotative programming.
+
+-   [Haskell ecosystem requests](https://www.snoyman.com/blog/2018/02/haskell-ecosystem-requests)
+
+    > A single comment in a mega-thread on GitHub is hardly a good place to write down these requests, however, and it seems like there's no progress on them. I'm going to instead put down these ideas here, with a bit more explanation, and a few more ideas that have popped up since then.
+
+-   [Atomic expressions generically](https://neilmitchell.blogspot.com/2018/02/atomic-expressions-generically.html)
+
+    > For certain hints HLint needs to determine if a Haskell expression is atomic. I wrote a generic method to generate expressions and test if they are atomic.
+
+-   [The magic "Just do it" type class](https://www.joachim-breitner.de/blog/735-The_magic_&#x201c;Just_do_it&#x201d;_type_class)
```

Comment:
Oops! This should not have ended up here. This post is from 2018-02-02 and was already featured in [issue 93](https://haskellweekly.news/issues/93.html). I'd have to guess that @nomeata doesn't mind the accidental double exposure, but I'm going to remove this nonetheless.